v3/src/components/web-view/web-view-inspector.tsx

```diff
@@ -30,7 +31,8 @@ export const WebViewInspector = observer(function WebViewInspector({tile, show}:
       webViewModel?.setUrl(url)
     }, {
       undoStringKey: "V3.Undo.webView.changeUrl",
-      redoStringKey: "V3.Redo.webView.changeUrl"
+      redoStringKey: "V3.Redo.webView.changeUrl",
+      log: logStringifiedObjectMessage("Change web view URL: %@", {url})
```

Comment:
Could use `logMessageWithReplacement` to just render the url without the braces.